operations/boot_orchestration/Rolling_Upgrades.md

```diff
@@ -0,0 +1,29 @@
+# Rolling Upgrades using BOS
+
+> **`NOTE`** This section is for BOS V2 only.
+
+> **`NOTE`** This feature is the replacement for CRUS, which was deprecated in CSM 1.2.0.
+
+BOS V2 allows users to stage a boot artifacts, configuration and operation such as a reboot.
+The workload manager can later trigger the operation through BOS to apply that staged information, allowing rolling updates when nodes have no job running on them.
+
+## Workflow
+
+1) A sysadmin configures the workload manager to call the `applystaged` endpoint of BOS, along with sending a payload containing the `xname` of the components to be operated on.  For more information on the endpoint and payload see [Applying a Staged State](./Stage_Changes_with_BOS.md#applying-a-staged-state)
+
+    > **`NOTE`** The boot artifacts and configuration staged with BOS will not be applied if the node is rebooted outside BOS.
+    This is because BOS is caching the staged boot information and configuration internally, but not updating BSS and CFS until it immediately before to boots or reboots the nodes.
```

Comment:
Reword:
until immediately before it boots or reboots the nodes.

Reply:
wow, I'm not sure my previous sentence was even English.